s4e-web/src/app/views/map-view/map-view.component.ts

```diff
@@ -97,7 +97,7 @@ export class MapViewComponent implements OnInit {
     this.availableDates$ = this.productQuery.selectAvailableDates();
   }
 
-  selectProduct (productId: number | null) {
+  selectProduct(productId: number | null) {
     console.log('selectProduct', productId);
```

Comment:
console logs should not usually go to the master (I know, that it's not always a rule...)